extensions/gmsa-dc/v1/Setup-gMSA.ps1

```diff
@@ -58,7 +59,19 @@ function ImportADPS()
         return $false
         }
     }
-	
+
+# Function to create a folder on Active Directory for file reading/writing test
+function CreateRemoteFolder()
+    {
+    try {
+        mkdir $RemoteFolder
+        New-SmbShare -Name "write_test" -Path $RemoteFolder -FullAccess "Enterprise Admins", "Domain Admins"
```

Comment:
Do you know if the default service accounts (`gmsa-e2e` in [this case](https://github.com/kubernetes-sigs/windows-testing/blob/f830ab4d04aa01e8ce9e8d68b1471ffb35a79c7e/extensions/gmsa-dc/v1/Setup-gMSA.ps1#L257)) have this level of access?

Reply:
I tried this out and it failed with access denied.  I added:

```
 Grant-SmbShareAccess -Name "write_test" -AccountName "gmsa-e2e$" -AccessRight Full -Force
```

Reply:
Ah I see, the command I used in my env was `New-SmbShare -Name "write_test" -Path "C:\write_test" -FullAccess "WebApp01Hosts"`. "WebApp01Hosts" is the security group created by domain admin and linked with the gmsa. But in this script, gmsa-e2e is not added to any security group so we need to grant the access to the service account.

I updated the code and could you help me test it in your env again? Thanks!